src/io/asyncsocket.c

```diff
@@ -715,8 +732,10 @@ static void on_connection(uv_stream_t *server, int status) {
             MVM_repr_push_o(tc, arr, msg_box);
             MVM_repr_push_o(tc, arr, tc->instance->boot_types.BOOTStr);
             MVM_repr_push_o(tc, arr, tc->instance->boot_types.BOOTInt);
+            MVM_repr_push_o(tc, arr, tc->instance->boot_types.BOOTInt);
```

Comment:
Why this addition?

Reply:
It's there as a placeholder for the listening socket's file descriptor from line 714

Having to wait until someone connects before its value can be known probably isn't the most ideal way of handling this, though